src/App.js

```diff
@@ -1,25 +1,163 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react';
+import styles from './App.module.css';
+import DateRangePicker from './components/DateRangePicker/DateRangePicker';
+import CalendarYearly from './components/Calendar/CalendarYearly';
+import CalendarQuarterly from './components/Calendar/CalendarQuarterly';
+import CalendarMonthly from './components/Calendar/CalendarMonthly';
+import EventList from './components/EventList/EventList';
+const faker = require('faker');
+
+const generatorEventList = (size=3) => {
+  const eventList = Array(size).fill().map((_,index)=>{
+    return (
+      {id:index+1, name:faker.git.commitMessage(), location: faker.address.streetName(), hostname:faker.name.firstName(), type:faker.random.boolean()?"Private":"Public", date:faker.date.between('2018-01-01', '2022-01-05')}
+    );
+  });
+  eventList.sort((a, b) => (a.date > b.date) ? 1 : -1)
+  return eventList;
+}
+
+class App extends Component {
+  state ={
+    eventList: generatorEventList(12),
+    startDate: new Date("2018-1-1"),
+    endDate: new Date("2022-12-31"),
+    yearLowerLimit:2018,
+    yearAmmountLimit:5,
+    period:"Yearly",
+    selectAll:true,
+    year:2020,
+    month1:0,
+    month2:11
+  };
```

Comment:
@edwartexe be aware of spaces and code readability
```suggestion
  state = {
    eventList: generatorEventList(12),
    startDate: new Date('2018-1-1'),
    endDate: new Date('2022-12-31'),
    yearLowerLimit: 2018,
    yearAmmountLimit: 5,
    period: 'Yearly',
    selectAll: true,
    year: 2020,
    month1: 0,
    month2: 11
  };
```